### PR TITLE
[4.0][a11y] Toolbar in a nav tag

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Enclose the toolbar in a navigation container. 

### Testing Instructions
Code inspection,
Listen to the Screenreader,
Check the site with AXE. 

### Expected result
Navigation is recogised as a landmark 
For sighted users there is no difference.

### CODE REVIEW

**Comment by zwiastunsw** [comment]
Please, remove the aria-label attribute from div tag, because you used the aria-label in the nav tag.

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 64de0fdec1156fbf3eae455146d58f321035e2d8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24013">issues.joomla.org/tracker/joomla-cms/24013</a>.</sub>

File: layouts/joomla/toolbar/containeropen.php [hunk 1]
```diff
@@ -14,4 +14,5 @@
 
 HTMLHelper::_('script', 'legacy/toolbar.min.js', array('version' => 'auto', 'relative' => true));
 ?>
-<div class="btn-toolbar d-flex" role="toolbar" aria-label="<?php echo Text::_('JTOOLBAR'); ?>" id="<?php echo $displayData['id']; ?>">
+<nav aria-label="<?php echo Text::_('JTOOLBAR'); ?>">
```
Comment (Quy) [inline layouts/joomla/toolbar/containeropen.php]:
```suggestion
<nav aria-label="<?php echo Text::_('JTOOLBAR'); ?>">
```

**Comment by wojsmol** [comment]
I have tested this item :white_check_mark: successfully on 946e4e8a3d712ecc2576c497a9d83baffbcf9e9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24013">issues.joomla.org/tracker/joomla-cms/24013</a>.</sub>

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 946e4e8a3d712ecc2576c497a9d83baffbcf9e9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24013">issues.joomla.org/tracker/joomla-cms/24013</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24013">issues.joomla.org/tracker/joomla-cms/24013</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**